### PR TITLE
New explore page

### PR DESCRIPTION
**Pull request by diogob**
Extracted from PR #55

### CODE REVIEW

File: src/contribution/projects-explore.js [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                    if (filter.filter === nearMe){
+                        project.getPageWithToken(nearMe.parameters()).then(vm.projectCollection);
+                    } else {
+                        project.getPage(filter.filter.parameters()).then(vm.projectCollection);
+                    }
+                    vm.toggleCategories();
+                },
+
+                toggleCategories: () => {
+                    document.getElementById('categories').classList.toggle('closed');
```
Comment (vicnicius) [inline src/contribution/projects-explore.js]: We should use m.prop to achieve this. : )

File: src/contribution/projects-explore.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                category: m.prop(),
+
+                loadProjects: (title, filter, category) => {
+                    vm.title(title);
+                    vm.category(category);
+                    projects.firstPage(filter.parameters());
+                    vm.toggleCategories.toggle();
+                },
+
+                loadRoute: () => {
```
Comment (vicnicius) [inline src/contribution/projects-explore.js]: Just to know, have you considered using mithril's routing here?

Comment (diogob) [inline src/contribution/projects-explore.js]: Yes, but I'm still not sure on how to use then for this situation. We can definitely chat about this.

**Comment by vicnicius** [comment]
:beers: Are we ready to merge?

**Pull request merged**